Podfile

```diff
@@ -21,3 +20,13 @@ end
 target 'ExamplesUITests' do
   # Pods for testing
 end
+
+post_install do |installer|
+    installer.pods_project.targets.each do |target|
+        if target.name == 'DocsCode'
+            target.build_configurations.each do |config|
+                config.build_settings['SWIFT_VERSION'] = '3.0'
```

Comment:
The DocsCode targets should use the same Swift version as the rest of the project → #123.